src/questions.js

```diff
@@ -0,0 +1,108 @@
+const questions = [
+  {
+    type: "input",
```

Comment:
Please add some validation to your questions. Currently I can just hit enter and leave them all blank.